_posts/2017-10-09-omero-5-4-0.md

```diff
@@ -0,0 +1,87 @@
+---
+layout: post
+title: Release of OMERO 5.4.0
+intro-blurb: The OME team is pleased to announce the release of OMERO 5.4.0.
+---
+Today we are releasing OMERO 5.4.0, a full production-ready release of OMERO;
+featuring a new configurable user role "Restricted Administrators"; further
+improvements to OMERO.web; additions to OMERO.cli; and many fixes and
+performance improvements:
+
+-  added [Administrators with restricted privileges](https://docs.openmicroscopy.org/omero/5.4.0/sysadmins/admins-with-restricted-privileges.html) to allow sysadmins to delegate
+   management tasks to facility managers without granting them full system
+   admin privileges, or to allow trusted users such as image analysts to carry
+   out tasks on behalf of all other users
+-  fixed color conversion to RGBA
+-  added support for exporting images in a plate as OME-TIFF
+-  improved creation of rendering settings for images without any stats e.g.
+   32bit images
+-  improved performance for moving large Plates
+-  fixed import bug in OMERO.insight
```

Comment:
The one I was thinking of is more a restriction when used with light admin role
https://github.com/openmicroscopy/openmicroscopy/pull/5468
The other one was already included in 5.3